cfn-resources/stream-connection/test/stream-connection-sample-cfn-request.json

```diff
@@ -0,0 +1,15 @@
+{
+  "desiredResourceState": {
+    "Profile": "maastha-cloud-dev",
+    "ProjectId": "65df4d741a7df1165f4ca4e2",
```

Comment:
I think we should remove the project id value here

Reply:
nice catch! removed